WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameViewController.swift

```diff
@@ -29,10 +29,28 @@ class SiteNameViewController: UIViewController {
         setTitleForTraitCollection()
     }
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        SiteCreationAnalyticsHelper.trackSiteNameViewed()
```

Comment:
We follow this pattern in other steps, but IMO `viewDidAppear` feels like a better fit.